server/startup/migrations/v219.js

```diff
@@ -0,0 +1,10 @@
+import { Migrations } from '../../../app/migrations';
+import { Settings } from '../../../app/models';
+
+Migrations.add({
+	version: 219,
```

Comment:
I'm pretty sure this will clash with new migrations 😬

Reply:
Oh my 🙈 sry @aditya-kumawat , I just merged another PR with migration `v223` 😢

Reply:
@sampaiodiego You also tagged wrong Aditya here xD

Reply:
damn github suggestions.. sry the confusion guys 😢

Reply:
Okay, no problem.
Added migration `v224`